test/testlib/minikube_utilities.go

```diff
@@ -547,3 +560,30 @@ func UnmarshalJSONArray(t *testing.T, stringJSON string) []interface{} {
 	assert.NilError(t, err)
 	return results
 }
+
+func VerifyAdminKvSetAndGet(t *testing.T, podName string, namespaceName string) {
+
+	options := k8s.NewKubectlOptions("", "")
+	options.Namespace = namespaceName
+
+	// verify the KV store can write and read in a reasonable time
+	start := time.Now()
+	output, err := k8s.RunKubectlAndGetOutputE(t, options,
+		"exec", podName, "--", "nuocmd", "set", "value", "--key", "test/minikube", "--value", "testVal", "--unconditional",
+	)
+	assert.NilError(t, err, "Could not set KV value")
+	elapsed := time.Since(start)
+
+	assert.Check(t, elapsed.Seconds() < 2.0, fmt.Sprintf("KV set took longer than 2s: %s", elapsed))
```

Comment:
why do you wanna check that it takes less than 2 sec? Is there some bug that you are trying to find? What does the failure of the test tell us?

I think that it might just tell us that the test VM was slow.

Reply:
Two things I have observed:
1. KV set/get is slooooow.
Even on our Rancher rig, I was seeing ~1s elapsed per operation.

2. Under some - as yet undetermined - conditions, the New Admin Raft gets brain-staggeringly slow:
~10-20secs to return from an operation - including KV set/get.

The new `nuosm` performs at least 10 KV get/set operations.
If they are regularly going to take >2 sec each, then we will hit timeouts with SMs starting - simply due to the time taken by KV get/set operations.

So this test is supposed to flag the problem with slooooow KV, so we don't waste time hunting down SM/TE timeouts that are actually being caused by Raft running through treacle.

Reply:
Got it. That makes sense! Thanks. Resolving.